bin/nyc.js

```diff
@@ -50,7 +51,7 @@ if (argv._[0] === 'report') {
     NYC_CONFIG: JSON.stringify(config),
     NYC_CWD: process.cwd(),
     NYC_ROOT_ID: nyc.rootId,
-    BABEL_DISABLE_CACHE: 1,
+    BABEL_DISABLE_CACHE: config['babel-cache'],
```

Comment:
Good suggestion, though I'd flip the boolean there so that the cache is defaulted to off.